KsApi/models/graphql/adapters/Project+ProjectFragment.swift

```diff
@@ -7,7 +7,8 @@ extension Project {
   static func project(
     from projectFragment: GraphAPI.ProjectFragment,
     rewards: [Reward] = [],
-    addOns: [Reward]? = nil
+    addOns: [Reward]? = nil,
+    backing: Backing? = nil
```

Comment:
So in this case, you've made `backing` part of the `ProjectFragment`, in which case we wouldn't need to pass it into this function because it's already in the fragment. If we were to only fetch it in specific queries then we would add it as an argument here.

Reply:
I see, I will create a backing from the `ProjectFragment` instead of passing it in.

Reply:
My only question is if I use `...BackingFragment` it will self reference `...ProjectFragment` which will result in a infinite loop, so instead of `...BackingFragment`, I'll add all the properties inside the fragment to project fragment's `backing { ... }` except `project` if that makes sense.

Reply:
If you have a minute to pair on this, would appreciate it!

Reply:
Ok, so revisiting your original comment, it seems like not making `backing` part of the fragment is the way to go. This static func is being called by `static func envelopeProducer(from data: GraphAPI.FetchBackingQuery.Data)` which has made the backing query already, which contains the `project`, so we can just pass it in here and remove `backing` from the `project fragment`.